vnext/Microsoft.ReactNative/LayoutService.idl

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import "IReactContext.idl";
+
+#include "NamespaceRedirect.h"
+#include "DocString.h"
+
+namespace Microsoft.ReactNative
+{
+  [default_interface]
+  [webhosthidden]
+  DOC_STRING("Provides access to Yoga layout functionality.")
+  runtimeclass LayoutService {
+    DOC_STRING("Use this method to get access to the @LayoutService associated with the @IReactContext.")
+    static LayoutService FromContext(IReactContext context);
+
+    DOC_STRING("Applies layout to all root nodes.")
+    void ApplyLayout();
+
+    DOC_STRING("Applies layout from a given React node.")
```

Comment:
these operations are recursive, I reckon? if so, worth clarifying in the doc string?

Reply:
And I think I'll eliminate the unconstrained ApplyLayout overload, I was adding that for the future ability to nest views inside text, but that is an internal API and doesn't need an ABI interface.